products/bash/release-ar.sh

```diff
@@ -30,7 +30,7 @@ function usage() {
 namespace="cp4i"
 release_name="demo"
 assetDataVolume="ibmc-file-gold-gid"
-couchVolume="ibmc-block-gold"
+couchVolume="cp4i-block-performance"
```

Comment:
Do we want `cp4i-block-performance` as default? If this script was used standalone, there's no docs for using the `-c` parameter to change this, and if they haven't run the 1-click they don't seem likely to have this storage class available!

Reply:
Sure, I'll revert that back. I doubt the default is used except by us when testing the scripts standalone, but fair enough! Though I'll add docs about the missing parameters

Reply:
Done!